config/default/language/fr/webform.webform.contact.yml

```diff
@@ -4,10 +4,5 @@ settings:
 handlers:
   email_confirmation:
     label: 'Courriel de confirmation'
-    settings:
```

Comment:
It seems odd that the translation files are being adjusted. Is this expected?

Reply:
I wasn't too sure about that, it may be due to enabling of translation for some fields, but will check that out.